preview-src/index.ts

```diff
@@ -145,57 +194,28 @@ function renderTitle(pr: PullRequest): HTMLElement {
 	return titleContainer;
 }
 
-function renderDescription(pr: PullRequest): HTMLElement {
-	const commentContainer = document.createElement('div');
-	commentContainer.classList.add('description-container');
-
-	const commentHeader = document.createElement('div');
-	commentHeader.classList.add('description-header');
-
-	const commentBody = document.createElement('div');
-	commentBody.className = 'comment-body';
-	commentBody.innerHTML = pr.bodyHTML ?
-		pr.bodyHTML :
-		pr.body
-			? md.render(emoji.emojify(pr.body))
-			: '<p><i>No description provided.</i></p>';
-
-	if (pr.labels.length) {
-		const line = document.createElement('div');
-		line.classList.add('line');
-
-		line.innerHTML = `<svg class="octicon octicon-tag" viewBox="0 0 14 16" version="1.1" width="14" height="16">
-			<path fill-rule="evenodd" d="M7.685 1.72a2.49 2.49 0 0 0-1.76-.726H3.48A2.5 2.5 0 0 0 .994 3.48v2.456c0 .656.269 1.292.726 1.76l6.024 6.024a.99.99 0 0 0 1.402 0l4.563-4.563a.99.99 0 0 0 0-1.402L7.685 1.72zM2.366 7.048a1.54 1.54 0 0 1-.467-1.123V3.48c0-.874.716-1.58 1.58-1.58h2.456c.418 0 .825.159 1.123.467l6.104 6.094-4.702 4.702-6.094-6.114zm.626-4.066h1.989v1.989H2.982V2.982h.01z" />
-			</svg>
-			${pr.labels.map(label => `<span class="label">${label}</span>`).join('')}`;
-
-		commentContainer.appendChild(line);
-	}
-
-	commentContainer.appendChild(commentHeader);
-	commentContainer.appendChild(commentBody);
-
-	if (pr.canEdit) {
-		function updateDescription(text: string) {
+function renderDescription(pr: PullRequest): void {
+	const descriptionNode = document.getElementById('description');
+	descriptionNode.innerHTML = '';
+	const bodyHTML = !pr.body ? '<i>No description provided</i>' : pr.bodyHTML;
+	const descriptionElement = renderComment({
```

Comment:
is this intentional that now we render the description as a comment? I kind of like the ui before.

Reply:
Do you mean having the information about the merge as the title of the description?
<img width="1042" alt="screen shot 2019-02-14 at 2 53 05 pm" src="https://user-images.githubusercontent.com/3672607/52823067-8c4b7c00-3068-11e9-99da-1c94dc5a3556.png">
<img width="1030" alt="screen shot 2019-02-14 at 2 54 11 pm" src="https://user-images.githubusercontent.com/3672607/52823102-a4230000-3068-11e9-9cb4-db0a40138f26.png">